### ISSUE TITLE
question: a strange issue on publishing module

### ISSUE THREAD

**Issue by yorkie** [npm/npm#9579]
Hey When I initialized and published one module to npm, then I got a weird log:

``` sh
$ npm publish
npm ERR! registry error parsing json
+ zhuge-node@1.0.0
```

https://www.npmjs.com/package/zhuge-node is working as well....even though it's not a problem, but still wanna put this here to let you know the potential issue maybe :)

**Comment by garrows** [comment]
Getting the same the last week or so with `npm@2.14.2`

**Comment by austinking** [comment]
I'm having a similar issue.

I was able to publish `https://github.com/austinking/upgrade-private-app` using npm@2.11.3.

Then I fixed a bug, bumped the package version number and re-published with

	npm --registry "http://registry.npmjs.org/" publish
	npm ERR! registry error parsing json
	npm ERR! registry error parsing json
	npm ERR! publish Failed PUT 403
	npm ERR! Darwin 14.5.0
	npm ERR! argv "node" "/usr/local/bin/npm" "--registry" "http://registry.npmjs.org/" "publish"
	npm ERR! node v0.12.7
	npm ERR! npm  v2.11.3
	npm ERR! code E403

	npm ERR! "You cannot publish over the previously published version 0.2.4." : upgrade-private-app
	npm ERR!
	npm ERR! If you need help, you may report this error at:
	npm ERR!     <https://github.com/npm/npm/issues>

	npm ERR! Please include the following file with any support request:
	npm ERR!     /Users/austinking/src/upgrade-private-app/npm-debug.log
	upgrade-private-app aok$ npm --version
	2.11.3

The command was very slow to display output, unlike the first publish.
